api/renter.go

```diff
@@ -103,14 +103,18 @@ func (srv *Server) renterAllowanceHandlerPOST(w http.ResponseWriter, req *http.R
 	// 	writeError(w, "Couldn't parse renewwindow: "+err.Error(), http.StatusBadRequest)
 	// 	return
 	// }
+	renewWindow := period / 4
```

Comment:
I would prefer period/2, because there's not much cost to doing it that way, and it means that people can forget about Sia for up to 6 weeks at a time. 6 weeks is a period used by most private bittorrent trackers, and it seems to work pretty well for them.

If it were only 3 weeks I'd worry that people would forget and lose their files.